nx.json

```diff
@@ -39,8 +39,13 @@
       }
     },
     "nx-release-publish": {
-      "dependsOn": ["build"],
-      "executor": "@nx/js:release-publish",
+      "cache": false,
+      "dependsOn": [
+        {
+          "target": "build",
+          "params": "forward"
+        }
```

Comment:
is needed to pass `--skip-nx-cache` flag down to the `nx-release-publish` task pipeline.